client/views/Footer.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Grid } from 'react-bootstrap'
+
+export default class Footer extends React.Component {
+  render () {
+    return (
+      <Grid className='page-footer'>
```

Comment:
Why is this a Grid if you are not using Col elements? I believe that div would be the same.